portal/templates/flask_user/emails/truenth_forgot_password_message.html

```diff
@@ -0,0 +1,23 @@
+{% extends 'flask_user/emails/base_message.html' %}
+
+{% block message %}
+
+<p>It sounds like you're having trouble logging in to {{ app_name }}.</p>
```

Comment:
Could you use the config `USER_APP_NAME` here instead of `app_name`?

It looks like `app_name` is set by the config `PROJECT`, and defaults to "portal", which makes me think `PROJECT` was a hacky way to get the [name of the project](https://github.com/uwcirg/true_nth_usa_portal/blob/develop/setup.py#L13), [a long time ago...](https://github.com/uwcirg/true_nth_usa_portal/commit/ae377588fc79d94ce54481e68d1e7d07995177fc#diff-b9ff4c6d3e8ce65648f66d4f5cef9fd6L9)